R/query.R

```diff
@@ -65,106 +65,56 @@ addClause <- function(query, keys, type = "FILTER", min = NULL, max = NULL, cate
     keys <- as.list(keys)
   }
 
-  variablesToAdd <- lookupVariables(query, keys)
-
   if(toupper(type) == "FILTER") {
     if (length(keys) != 1) {
       message("Filters must be added one at a time")
       return (query)
     }
-    if (nrow(variablesToAdd) == 0) {
-      variablesToAdd <- lookupGenomicVariables(query, keys)
-      if (nrow(variablesToAdd) == 0) {
-        message("Variables not found:")
-        message(keys)
+
+    if(typeof(categories) == "list") {
+        query$categoryFilters[[keys[[1]]]] <- categories
```

Comment:
I hate this keys thing but I have no better suggestions. lol

Reply:
Yeah, this R "object" is very odd because our JSON format is not normal, and jsonlite is not very flexible